Documentation/cdsIn.md

```diff
@@ -0,0 +1,11 @@
+﻿# CdsWebHook - C<span>#</span>
+
+The `CdsWebHook` makes it incredibly easy to have your CDS functions executed via an HTTP call to your function.
```

Comment:
Is WebHook the right term? I'm a bit ignorant about web-hooks, wo please educate me what they are, and if this template is in fact a web-hook.